Added 3 small hacks:
* Fix the path of the icon images
* Remove bgcolor, border-right-color and border-bottom-color attributes, they
  are not handled by fop
* Handle dejavu fonts
diff -Naur asciidoc-8.2.3.orig/a2x asciidoc-8.2.3/a2x
--- asciidoc-8.2.3.orig/a2x	2007-07-18 10:09:06.000000000 +0200
+++ asciidoc-8.2.3/a2x	2007-11-07 23:12:33.000000000 +0100
@@ -513,8 +513,10 @@
     pdf="$DESTINATION_DIR/$SRC_NAME.pdf"
     to_docbook
     execute_command_2 "xsltproc $XSLTPROC_OPTS --nonet \
-        \"$xsl\" \"$xml\" >\"$fo\""
-    execute_command_2 "fop.sh \"$fo\" \"$pdf\""
+       \"$xsl\" \"$xml\" \
+       | sed 's|\(src=\"./images/icons/[^\"]\+\).png\"|\1.jpg\"|g' \
+       | sed 's/bgcolor=\"[^\"]\+\"//g;s/border-right-color=\"\"//g;s/border-bottom-color=\"\"//g' >\"$fo\""
+    execute_command_2 "fop.sh -c /usr/share/fop/dejavu-ttf.xml \"$fo\" \"$pdf\""
 }
 
 function to_odt()
